pkgs/development/tools/misc/texinfo/texinfo-cross.patch

```diff
@@ -0,0 +1,29 @@
+--- texinfo-6.5/configure.ac.orig	2018-11-30 07:00:11.027337595 +0300
++++ texinfo-6.5/configure.ac	2018-11-30 07:28:03.764954462 +0300
+@@ -91,7 +91,8 @@
+ gl_INIT
+ 
+ # help2man to generate man pages.
+-AM_MISSING_PROG(HELP2MAN, help2man)
++#AM_MISSING_PROG(HELP2MAN, help2man)
++AC_PATH_PROG([HELP2MAN], [help2man])
+ 
+ AC_PROG_INSTALL
+ AC_PROG_MAKE_SET
+@@ -212,10 +213,12 @@
+   # env -i gives this build host configure a clean environment;
+   # consequently, we have to re-initialize $PATH.
+   env -i CC="$BUILD_CC" AR="$BUILD_AR" RANLIB="$BUILD_RANLIB" \
```

Comment:
How about removing `env -i` here instead?

Reply:
Not a good idea.

Reply:
It would be better to add `NIX_BUILD_CFLAGS_COMPILE=$NIX_BUILD_CFLAGS_COMPILE` and `NIX_BUILD_LDFLAGS=$NIX_BUILD_LDFLAGS` to that list, and maybe also `LD=$BUILD_LD`.